tools/ci-ts/fixtures/flux-monitor-job.ts

```diff
@@ -18,7 +18,7 @@ export default {
           disabled: true,
         },
         pollTimer: {
-          period: '5s',
+          period: '10s',
```

Comment:
What was this for?

Reply:
It seemed like this test would occasionally flake due to the poll timer being too fast, and 10s didn't slow the test down measurably.  If you want, I can try it again with a `5s` timer.